src/numbuf/thirdparty/download_thirdparty.sh

```diff
@@ -13,4 +13,4 @@ fi
 cd $TP_DIR/arrow
 git pull origin master
 
-git checkout 670612e6fdf699486641ed0d39d22257eb8acdb2
+git checkout 8a700ccdad745c250fe5d91a9104e7c2d6364c1b
```

Comment:
It probably doesn't matter, but this isn't exactly the commit tagged for the 0.4.0 release. I believe that one is a8f8ba0cbcf5f596f042e90b7a208e7a0c3925b7.